deno.jsonc

```diff
@@ -38,7 +38,6 @@
     "dnt": "deno run -A _tasks/dnt.ts",
     "test": "deno task ctx -- deno test -A -r -L=info --ignore=target --parallel",
     "test:update": "deno task test -- --update",
-    "moderate": "deno run -A https://deno.land/x/moderate@0.0.5/mod.ts && dprint fmt",
```

Comment:
?

Reply:
Didn't we decide to get rid of the `mod.ts` in examples?

Reply:
Yes, but we use `moderate` elsewhere